tests/crontime.test.ts

```diff
@@ -213,6 +213,11 @@ describe('crontime', () => {
 		expect(() => {
 			new CronTime('* * * 1/0 *');
 		}).toThrow();
+
+		// infinite loop when providing two or more zeros as step (#742)
```

Comment:
should this comment be changed? we're testing because we want to avoid there being an infinite loop

Reply:
I'm not sure what you mean. the comment is the name of the issue (#742), but maybe this would be better rephrased. any idea?
the idea was to reference the issue in case someone needs to understand why this test exists.

Reply:
I would just change it to something like "throw in case of malformed input" or something, idk. I know in our specific case this happened to cause an infinite loop but that's because of the way the code was written. regardless we should throw an error if we receive that kind of input. the test case really is preventing invalid input, not preventing infinite loops specifically

Reply:
that makes absolute sense, I'll simply remove the comment altogether